.github/workflows/ci.yml

```diff
@@ -17,21 +17,32 @@ on:
       - '**/*.md'
       - '.prettierrc'
       - '**/*ignore'
+  workflow_call:
+
+env:
+  NODE_VERSION: '20'
 
 jobs:
   test:
-    name: Unit + Type Check / Node.js ${{ matrix.node }}
-    runs-on: ubuntu-latest
+    name: Unit + Type Check / Node.js ${{ matrix.node }} / OS ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node: [20]
+        node: [20, 22, 24]
```

Comment:
I think a lowest common denominator of Node versions (i.e. just 20) is fine tbh - can't recall the last time something broke because of a Node version upgrade